### ISSUE TITLE
(signal) NoAutoUpdate

### ISSUE THREAD

**Issue by majkinetor** [majkinetor/au-packages#136]
Must be done via hosts file

`127.0.0.1 updates.signal.org`

**Comment by 360path** [comment]
I think having `127.0.0.1 updates.signal.org` in the hosts file doesn't prevent Signal from searching/finding updates. I keep geting a blue update bar in the Signal Desktop App whenever an update is available even though I have that line in my hosts file.

**Comment by majkinetor** [comment]
Yeah, the option suck and shouldn't be used because of unfortunate choco problem with remembering arguments I believe. 

You should remove that hosts file option and I will remove no-autoupdate param.

**Comment by majkinetor** [comment]
I mean, it works great when it doesn't suck, its the only way I know to prevent Signal showing there are updates.

**Comment by 360path** [comment]
ok, I understand. Thank you!

**Comment by majkinetor** [comment]
I will reopen this to maybe inspect further or remove an option.
